Ports/perl5/patches/ReadMe.md

```diff
@@ -4,20 +4,11 @@
 
 pp_sys: Avoid redefining `h_errno`
 
-
-## `0002-Makefile-Don-t-run-lib-unicore-mktables-with-maketes.patch`
-
-Makefile: Don't run `lib/unicore/mktables` with -maketest
-
-This argument causes the interpreter to hang and consume huge amounts
-of RAM.
```

Comment:
Was this fixed by this PR (or more specifically, by undoing the changes to the random function configuration)? This looks to be a host command, so I can't imagine why it would suddenly work otherwise.

Reply:
When I re-enable the `drand48` patch to the configure script, I get a smörgåsbord of errors from the same script and I my system's RAM consumption spikes with a hang in `miniperl`.

Reply:
Errors:
```[perl5/build] Use of uninitialized value $spaces_before in concatenation (.) or string at lib/unicore/mktables line %.
[perl5/build] Use of uninitialized value $spaces_after in concatenation (.) or string at lib/unicore/mktables line %.
[perl5/build] Use of uninitialized value $parts[0] in join or string at lib/unicore/mktables line %.
[perl5/build] Use of uninitialized value $parts[1] in join or string at lib/unicore/mktables line %.
[perl5/build] Use of uninitialized value within @bad_loose_seps in concatenation (.) or string at lib/unicore/mktables line %.
[perl5/build] Use of uninitialized value $spaces_before in concatenation (.) or string at lib/unicore/mktables line %.
[perl5/build] Use of uninitialized value $spaces_after in concatenation (.) or string at lib/unicore/mktables line %.
[perl5/build] Use of uninitialized value $parts[0] in join or string at lib/unicore/mktables line %.
[perl5/build] Use of uninitialized value $parts[1] in join or string at lib/unicore/mktables line %.
[perl5/build] Use of uninitialized value $spaces_before in concatenation (.) or string at lib/unicore/mktables line %.
[perl5/build] Use of uninitialized value $spaces_after in concatenation (.) or string at lib/unicore/mktables line %.
[perl5/build] Use of uninitialized value $spaces_before in concatenation (.) or string at lib/unicore/mktables line %.
[perl5/build] Use of uninitialized value $spaces_after in concatenation (.) or string at lib/unicore/mktables line %.```